pkg/config/api.go

```diff
@@ -53,54 +62,85 @@ const (
 )
 
 var (
-	slots        = make(map[string]StoreOperate)
+	slots = make(map[string]StoreOperate)
+
 	storeOperate StoreOperate
+
+	once sync.Once
 )
 
-func GetStoreOperate() (StoreOperate, error) {
-	if storeOperate != nil {
-		return storeOperate, nil
+// Regis register store plugin
+func Regis(s StoreOperate) {
+	if _, ok := slots[s.Name()]; ok {
+		panic(fmt.Errorf("StoreOperate=[%s] already exist", s.Name()))
 	}
 
-	return nil, errors.New("StoreOperate not init")
+	slots[s.Name()] = s
 }
 
-func Init(name string, options map[string]interface{}) error {
-	s, exist := slots[name]
-	if !exist {
-		return fmt.Errorf("StoreOperate solt=[%s] not exist", name)
+type (
+	callback func(e Event)
+
+	SubscribeResult struct {
+		EventChan <-chan Event
+		Cancel    context.CancelFunc
 	}
 
-	storeOperate = s
+	subscriber struct {
+		watch callback
+		ctx   context.Context
+	}
 
-	return storeOperate.Init(options)
-}
+	Options struct {
+		StoreName string                 `yaml:"name"`
+		RootPath  string                 `yaml:"root_path"`
+		Options   map[string]interface{} `yaml:"options"`
+	}
 
-// Register register store plugin
-func Register(s StoreOperate) {
```

Comment:
why rename Register to Regis? I think maybe the Register is a better method name. Just discuss.